starters/vue3-xstate-css/src/views/GreetView.vue

```diff
@@ -0,0 +1,78 @@
+<script setup lang="ts">
+import { useMachine } from '@xstate/vue';
+import HeaderComponent from '@/components/HeaderComponent.vue';
+import { greetMachine } from '@/machines/greetMachine';
+
+const props = defineProps({
+  query: String,
+});
+
+const { state } = useMachine(greetMachine(props.query || ''), {
+  devTools: true,
+});
+</script>
+
+<template>
+  <main>
+    <HeaderComponent> Fetch Data from API </HeaderComponent>
+    <section class="fetch__section">
+      <p class="fetch__section-title">Message:</p>
+      <div class="fetch__section-result" data-cy="message-result">
+        <div
+          v-if="state.value === 'loading'"
+          class="fetch__section-loader"
+        ></div>
+        <p v-if="state.value === 'failure'" class="fetch__section-message-fail">
+          {{ state.context.error }}
+        </p>
+        <p v-else class="fetch__section-message-success">
+          {{ state.context.message }}
+        </p>
+      </div>
+    </section>
+    <div class="fetch__home-link">
+      <RouterLink to="/">Return Home</RouterLink>
+    </div>
+  </main>
+</template>
+
+<style scoped>
+.fetch__section {
```

Comment:
Not a request for change, I am just curious, if the styles are scoped why prefix the classes with `fetch__`?

You know they are scoped to the component.

Reply:
...mostly it just felt right? lol It's definitely not needed for the styles since they are scoped. But it's how my brain came up with the names. 🤷🏼 lol